provider/ansible.py

```diff
@@ -133,20 +133,13 @@ def python_install():
             LOG_JOB.error("Failed to get available pip binary")
             return False
         install_cmd = '%s install ansible' % pip_bin
-        # Install ansible depended packages that can't be installed
-        # automatically by pip when installing ansible
-        package_list = params.get_list("package_list", 'sshpass')
-        if package_list:
-            if not utils_package.package_install(package_list):
-                LOG_JOB.error("Failed to install '%s'.", package_list)
-                return False
         status, output = process.getstatusoutput(install_cmd, verbose=True)
         if status != 0:
             LOG_JOB.error("Install python ansible failed as: %s", output)
             return False
         return True
 
-    def distro_install():
+    def distro_install(package="ansible"):
         """
         Install ansible from the distro
```

Comment:
IMO, this function can be kept as the original approach.   Be used for ansible installation only.

Reply:
We still need it, see my previous comment, please :slightly_smiling_face: